### DISCUSSION TITLE
(taiko) cursor should be hidden based on cursor size, not just position

### DISCUSSION THREAD

**Discussion by chayleaf** [ppy/osu#16927] [Ideas] [answered]
Currently, in Taiko, the cursor gets hidden when hovering over the play area. For example, in the image below, if I were to move the cursor up, it gets hidden:

![image](https://user-images.githubusercontent.com/9590981/154833339-7697cb64-396c-4c9e-b9f2-e29b8c78cafc.png)

However, if we check the "cursor visibility border" at the top, it turns out the cursor can still overlap the playfield!

![image](https://user-images.githubusercontent.com/9590981/154833470-0d7cac59-3d6f-4521-bb4b-72949c4412f1.png)

Personally, that looks quite distracting to me. So, the top border should be offset by the cursor size.

**Comment by peppy** [comment, marked as answer]
The plan is to make the cursor fade out when not in use, even in such cases. I believe there's at least one other issue covering this.

**Reply by bdach** [reply]
Would be https://github.com/ppy/osu/issues/9523 I believe?

**Reply by peppy** [reply]
Seems about right

**Reply by frenzibyte** [reply]
I think we can track the issue there and mark this one as answered.
